### PR TITLE
Feat plant schema logics

### PR DESCRIPTION
**Pull request by Xemko**
Add some of basic plant logic, work in progress.

### CODE REVIEW

**Review by tgcaspa — APPROVED** [review]

**Comment by Xemko** [comment]
Changes error code to 401 per your request

File: apps/server/requests.rest [hunk 1]
```diff
@@ -19,4 +19,22 @@ Content-Type: application/json
     "name": "xem2",
     "email": "xem2@xem.test",
     "phoneNumber": "0546211007"
+}
+###
+POST http://localhost:3000/api/plants/
+Content-Type: application/json
+
+{
```
Comment (tgcaspa) [inline apps/server/requests.rest:27]: if the source if "plants" should be an array

File: apps/server/requests.rest [hunk 2]
```diff
@@ -19,4 +19,22 @@ Content-Type: application/json
...
+    "description": "Cactus is a plant",
+    "image": "placeholder",
+    "waterFrequency" : 1,
+    "userId": "657cf041a2328d1b60592bd8"
+}
+###
+GET http://localhost:3000/api/plants/
+Content-Type: application/json
+
+{
```
Comment (tgcaspa) [inline apps/server/requests.rest:38]: if the source if "plants" should be an array

Comment (tgcaspa) [reply apps/server/requests.rest:38]: if you want one -> /plants/:id

File: apps/server/src/controllers/plant.controller.js [hunk 3]
```diff
@@ -1,16 +1,25 @@
...
 
-const getPlants = async (req, res) => {
+const getPlant = async (req, res) => {
     try {
-        const userId = req.user.id;
+        const plantName = req.body.name;
         // TODO getPlantsByUserId
-        const plants = await plantService.getPlants(userId);
-        return res.status(200).json({code: 200, message: 'Success', data: plants});
+        const plant = await plantService.getPlant(plantName);
```
Comment (tgcaspa) [inline apps/server/src/controllers/plant.controller.js:7]: should be by user, I think

**Review by tgcaspa — APPROVED** [review]

**Pull request merged**